blog/2024-10-10-beyond-byoc-wasmcloud-everywhere-platform/index.mdx

```diff
@@ -0,0 +1,77 @@
+
+![Build applications with reusable blocks of functionality](./images/build.png)
+
+**wasmCloud** is built from the ground up to orchestrate components, much as Kubernetes is designed to orchestrate containers. In this way, you can think of it as a distributed **application control plane**, in contrast with Kubernetes, which really excels at abstracting and managing infrastructure but has always lacked a first-class way to represent and manage applications.
+
+Being Wasm-native means wasmCloud can take advantage of components' unique features to enable new patterns and possibilities. For example, multi-cluster Kubernetes has never really been satisfactorily solved, but for wasmCloud, multi-cluster patterns are just a typical Tuesday. 
+
+wasmCloud provides out-of-the-box load balancing, failover, and other features for systems that span across environments such as clouds, edges, on-prem datacenters, and indeed Kubernetes clusters, since [wasmCloud is happy to run standalone or in an org's existing Kubernetes](/docs/kubernetes).
+
+## A platform for platform engineering
+
+When you’re thinking about platform engineering, the two most involved personas are **developers** and **platform teams**. Component characteristics like composability and polyglot interoperability are game-changing for developers, but Wasm and wasmCloud are also transformative for platform teams. 
+
+If you need a vendor solution to run at the edge, on highly resource-constrained hardware, deploying via Kubernetes and containers may not be practical or even possible. But the portability of components and distributed nature of wasmCloud mean that solutions can be deployed and managed from anywhere.
+
+![Orchestrate applications on any cloud, Kubernetes, or edge, including your own](./images/orchestrate.png)
+
+Typically, enterprises don’t create platforms from whole cloth—they ***curate* platforms by combining pre-existing cloud native technologies** to form a bespoke, purpose-built platform. 
+
+In wasmCloud, we’re building a platform that enables different teams to collaborate on what will be their bespoke platform—and not just collaborate on the initial construction, but in ongoing maintenance. Teams can create **golden templates that remain pluggable after adoption**, dramatically simplifying maintenance in the long run. 
+
+Ultimately, these pieces come together to deliver the next step in flexibility and control for enterprises: **the BYOC model on any cloud, Kubernetes, or edge, including your own**. The benefits that make BYOC so appealing stand to be dramatically expanded by using wasmCloud as an everywhere platform:
+
+- **Security**: Secure-by-default components, [wasmCloud's Zero Trust design](https://www.notion.so/docs/hosts/security), and flexible [secrets support](https://www.notion.so/docs/concepts/secrets) provide a rock-solid foundation for satisfying data sovereignty concerns.
```

Comment:
Probably these links were for wasmCloud.com?

Reply:
bleh, thanks for the catch!